wp-content/themes/bh/scss/_responsive.scss

```diff
@@ -2820,6 +2871,15 @@
 
 @media (min-width: 481px) and (max-width: 1199px) {
 
+	/***************/
+	/* events list */
+	/***************/
+
+	.col-1.visible-xs {

```

Comment:
Critically wrong - lack of hierarchy might cause major errors in other sections.  
Structure here should be follow exactly the same structure of event.scss, hence the full hierarchy should be declared:
`.events-list {
  .row {
    .col-1 {
    ...`